From 124cef85d3a20f30a3411a9f42a8a478c47d3f63 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 27 Jan 2012 17:37:23 -0500
Subject: [PATCH] gudev: Ensure we use libtool to link

Otherwise we won't find the shared library which is hidden in .libs.
---
 Makefile.am |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 7a35395..c264398 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -392,6 +392,7 @@ extras/gudev/GUdev-1.0.gir: extras/gudev/libgudev-1.0.la $(G_IR_SCANNER)
 		--library=gudev-1.0 \
 		--library-path=$(top_builddir)/libudev \
 		--library-path=$(top_builddir)/extras/gudev \
+		--libtool="$(LIBTOOL)" \
 		--output $@ \
 		--pkg=glib-2.0 \
 		--pkg=gobject-2.0 \
-- 
1.7.6.5

